From patchwork Fri Jun 26 09:02:06 2015
Content-Type: text/plain; charset="utf-8"
MIME-Version: 1.0
Content-Transfer-Encoding: 8bit
Subject: [v3, 3/7] mfd: intel_soc_pmic_crc: Add PWM cell device for Crystalcove
	PMIC
From: "Kumar, Shobhit" <shobhit.kumar@intel.com>
X-Patchwork-Id: 6679181
Message-Id: <1435309330-50725-4-git-send-email-shobhit.kumar@intel.com>
To: linux-pwm <linux-pwm@vger.kernel.org>,
	intel-gfx <intel-gfx@lists.freedesktop.org>,
	linux-kernel <linux-kernel@vger.kernel.org>,
	dri-devel <dri-devel@lists.freedesktop.org>,
	linux-gpio <linux-gpio@vger.kernel.org>
Cc: Alexandre Courbot <gnurou@gmail.com>, Paul Bolle <pebolle@tiscali.nl>,
	Samuel Ortiz <sameo@linux.intel.com>,
	Povilas Staniulis <wdmonster@gmail.com>, 
	Jani Nikula <jani.nikula@intel.com>,
	Shobhit Kumar <shobhit.kumar@intel.com>, 
	Linus Walleij <linus.walleij@linaro.org>, varkabhadram@gmail.com,
	Paul Gortmaker <paul.gortmaker@windriver.com>, bloften80@gmail.com,
	David Airlie <airlied@linux.ie>, Chih-Wei Huang <cwhuang@android-x86.org>, 
	Thierry Reding <thierry.reding@gmail.com>,
	Daniel Vetter <daniel.vetter@intel.com>, Lee Jones <lee.jones@linaro.org>
Date: Fri, 26 Jun 2015 14:32:06 +0530

Needed for PWM control suuported by the PMIC

CC: Samuel Ortiz <sameo@linux.intel.com>
Cc: Linus Walleij <linus.walleij@linaro.org>
Cc: Alexandre Courbot <gnurou@gmail.com>
Cc: Thierry Reding <thierry.reding@gmail.com>
Acked-by: Lee Jones <lee.jones@linaro.org>
Tested-by: Ville Syrjälä <ville.syrjala@linux.intel.com>
Signed-off-by: Shobhit Kumar <shobhit.kumar@intel.com>

---
v2: Correct the subject line (Lee jones)

 drivers/mfd/intel_soc_pmic_crc.c | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/drivers/mfd/intel_soc_pmic_crc.c b/drivers/mfd/intel_soc_pmic_crc.c
index 7436075..4a74948 100644
--- a/drivers/mfd/intel_soc_pmic_crc.c
+++ b/drivers/mfd/intel_soc_pmic_crc.c
@@ -109,6 +109,9 @@ static struct mfd_cell crystal_cove_dev[] = {
 	{
 		.name = "crystal_cove_pmic",
 	},
+	{
+		.name = "crystal_cove_pwm",
+	},
 };
 
 static const struct regmap_config crystal_cove_regmap_config = {
